### DISCUSSION TITLE
Missing count and percentile column in meta.feature

### DISCUSSION THREAD

**Discussion by Chhiring-Lama** [stuart-lab/signac#1086] [Q&A] [answered]
Hi, 

I am performing gene-peak linkage, but I am getting an error: 

`bm_1121_int <- LinkPeaks(object = bm_1121_int, peak.assay = "peaks",
 expression.assay = "SCT", genes.use = bm_1121_int@assays$SCT@var.features,
 pvalue_cutoff = 0.05, score_cutoff = 0.05)`

Error: 

`  |                                                  | 0 % ~calculating  Error in MatchRegionStats(meta.feature = meta.use, query.feature = pk.use[x,  : 
  count not present in meta.features`

When I checked the chromatin assay, 'count' and 'percentile' columns were not present. What would be the reason and is there a way to add them?

**Comment by timoast** [comment, marked as answer]
This is computed when you run `FindTopFeatures()`

I just pushed an update to the develop branch so that these metadata are computed automatically if they're missing

**Reply by Chhiring-Lama** [reply]
Thanks a lot!
